.github/tracing/README.md

```diff
@@ -48,4 +48,49 @@ This folder contains the following config files:
 
 These config files are for an OTEL collector, grafana Tempo, and a grafana UI instance to run as containers on the same network.
 `otel-collector-dev.yaml` is the configuration for dev (i.e. your local machine) environments, and forwards traces from the otel collector to the grafana tempo instance on the same network. 
-`otel-collector-ci.yaml` is the configuration for the CI runs, and exports the trace data to the artifact from the github run. 
\ No newline at end of file
+`otel-collector-ci.yaml` is the configuration for the CI runs, and exports the trace data to the artifact from the github run.
+
+## Instrumenting Custom Traces
+
+Adding custom traces requires identifying an observability gap in a related group of code executions. This is intuitive for the developer:
+
+- "What's the flow of component interaction in this distributed system?"
+- "What's a ballpark latency of funcOne?"
+- "What's the behavior of the JobProcessorOne component when jobs with [x, y, z] attributes are processed?"
+
+Once an execution flow is desired to be traced, the developer will decide which parts of the execution flow should be measured separately. Each logically separate measure will be a span. Spans can have one parent span and multiple children span. The relations between parent and child spans will make a directed acyclic graph.
+
+The most trivial application of a span is measuring latency performance at high levels of granularity (individual func executions). There is much more you can do, including creating human readable and timestamped events within a span (useful for monitoring concurrent access to resources), recording errors, linking parent and children spans through large parts of an application, and even extending a span beyond a single process.
```

Comment:
are there otel best practices we can link to? you don't wan the reader to interpret this as a suggestion to always trace every func. that's a lot of manual work. presumably there is an 80/20 rule wrt to return on investment of manual instrumentation. maybe otel or some other reputable source has guidelines

Reply:
I searched for a primary-source from OTEL on best practices but didn't find anything. This is a good 3rd party source: https://www.zerok.ai/post/distributed-tracing-best-practices